meta-schema.graphql

```diff
@@ -0,0 +1,115 @@
+# Base types
+type Party {
+	id:ID!
+	idType:String! #PartyId, IPI, ISNI, MBID
+	displayName:String!
+	firstName: String
+	lastName: String
+}
+
+type Organisation {
+	id:String!
+	idType:String! #DPid, IPI, MBID
+	displayName:String!
+	formalName:String
+}
+
+# Music Types
+type TerritoryAgreement {
+	territoryCode:Int! #CISAC Territory standard
```

Comment:
Should we have a `Territory` type which has a `code` but also `displayName` for displaying in the UI? So for example:

```graphql
type TerritoryAgreement {
  territory: Territory
}

type Territory {
  code:        Int
  displayName: String
}
```

Reply:
I thought it was surplus at this level. By ISO territory codes it ensures consistency of data for validation and keeps the footprint light. The mapping into more UI friendly terms can be done at the client level.

Reply:
> The mapping into more UI friendly terms can be done at the client level.

Hmm I think it's nice if the data has all the details. As for the footprint, the client only gets back what they request, so they can choose to omit the `displayName` if they don't want it.

Your call anyway.